models/product-type/src/type-definition/transformers.ts

```diff
@@ -0,0 +1,26 @@
+import { LocalizedString } from '@commercetools-test-data/commons';
+import { Transformer } from '@commercetools-test-data/core';
+import type { TTypeDefinition, TTypeDefinitionGraphql } from './types';
+
+const transformers = {
+  default: Transformer<TTypeDefinition, TTypeDefinition>('default', {
+    buildFields: ['fieldDefinitions', 'name'],
+  }),
+  rest: Transformer<TTypeDefinition, TTypeDefinition>('rest', {
+    buildFields: ['fieldDefinitions', 'name'],
+  }),
+  graphql: Transformer<TTypeDefinition, TTypeDefinitionGraphql>('graphql', {
+    buildFields: ['fieldDefinitions'],
+    replaceFields: ({ fields }) => {
+      const nameAllLocales = LocalizedString.toLocalizedField(fields.name);
+
```

Comment:
Should we transform the field `name` using `LocalizedString.resolveGraphqlDefaultLocaleValue` ?

[For example](https://github.com/commercetools/test-data/blob/a3ccfca0a4cc94eb6bcae6ccba050a9999c8f341/models/attribute-group/src/transformers.ts#L19)

Reply:
Updated here: 63053ce8aeda6ddae51dcf9aea3bd837d03ce565